src/components/Breed/BreedButton.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+import classes from "./BreedList.module.css";
```

Comment:
Try to use StyledComponents as everybody is using it and it will be good for you to get use to it. But, there is nothing wrong with this approach